src/nvme/mi-mctp.c

```diff
@@ -696,6 +696,13 @@ nvme_mi_ep_t nvme_mi_open_mctp(nvme_root_t root, unsigned int netid, __u8 eid)
 	return NULL;
 }
 
+__u8 nvme_mi_mctp_get_eid(nvme_mi_ep_t ep)
+{
+	struct nvme_mi_transport_mctp *mctp;
+	mctp = ep->transport_data;
```

Comment:
@jk-ozlabs , What is the best way to do this check?

Reply:
I'd say something like:

```c
    if (ep->transport == &nvme_mi_transport_mctp)
```

\- but we'll also need to think about how we're handing non-MCTP transports here; even if that's fairly hypothetical at the moment. Mainly: what sort of CSI conflicts are we checking for here? What does it mean if two `ep`s have different transports?

Perhaps this function could return a transport-specific identifier, rather than specifically an EID. For MCTP's case, that identifier could be the EID.